### DISCUSSION TITLE
How to delete historical data for mini-graph-cards

### DISCUSSION THREAD

**Discussion by MichiW1** [kalkih/mini-graph-card#737] [Q&A] [answered]
Hey all,
I like the mini-graph-cards a lot to visualize my energy and water consumption daily in bar graphs.

However, recently my water meter did heavy readout-errors and pushed values with several thousands of liters forward :-( 
I have corrected the readout, it is accurate again, but I have remaining bars in my bargraph with extremely high values which are not correct.

How can I delete these wrong history values or the whole history?
I have already deleted the history of the correlated HAS sensor in states.db but it seems that mini-graph has its own storage of history which I couldn't find and correct so far...

Can you help mme?

Thanks a lot,
Michael

**Comment by ildar170975** [comment, marked as answer]
Clean a browser’s cache

**Reply by MichiW1** [reply]
Hi @ildar170975 thanks for the hint, of course that was an easy one ;-) It works already well in my browser view.
However, I also need to find a way to delete the caches of my Android and iOS HAS apps, but I'll search for it...
Thanks a lot, Michael

**Comment by ildar170975** [comment]
I am not sure but at least iOS Companion app has a “reset frontend cache” button, may be it is same?
